pkg/analysis/passes/safelinks/safelinks.go

```diff
@@ -0,0 +1,120 @@
+package safelinks
```

Comment:
please add tests for this.

you will have to mock the webrisk api because we don't want to hit the API when running tests.